math.p8

```diff
@@ -1,24 +1,27 @@
-pico-8 cartridge // http://www.pico-8.com
-version 18
-__lua__
--- pico8lib math library
--- by sparr
+--- @module math
+--- Mathematical operations, mostly on numbers
+-- @author by sparr
 
 
-------------------------------------------------------------------------
--- lookup tables
```

Comment:
I think we might use `@section` for stuff like this, but I can make that switch after this is merged if you don't want to go through all the files again.

Reply:
ldoc doesn't like documenting module level tables assigned to variables. No idea why. Honestly the more I've used ldoc the less mature of a product it seems to be. Bugs everywhere, but it's the best we've got for lua right now (that I'm aware of)

Reply:
`@section` doesn't document a specific thing, it segregates the whole file into pieces that may or may not contain functions or other documented things.

On the topic of your reply, though... Maybe we add a function that just returns the table, and document that function? Users should be able to just copy the table out for their own use case.